src/main/java/ch/njol/skript/expressions/ExprDurability.java

```diff
@@ -36,40 +36,45 @@
 import ch.njol.skript.util.slot.Slot;
 import ch.njol.util.coll.CollectionUtils;
 
-/**
- * @author Peter Güttinger
- */
-@Name("Data/Damage Value")
-@Description({"The data/damage value of an item/block. Data values of blocks are only supported on 1.12.2 and below.",
-		"You usually don't need this expression as you can check and set items with aliases easily, ",
-		"but this expression can e.g. be used to \"add 1 to data of &lt;item&gt;\", e.g. for cycling through all wool colours."})
-@Examples({"set damage value of player's tool to 10",
-		"set data value of target block of player to 3",
-		"add 1 to the data value of the clicked block",
-		"reset data value of block at player"})
-@Since("1.2")
+@Name("Damage Value/Durability")
+@Description("The damage value/durability of an item.")
+@Examples({
+	"set damage value of player's tool to 10",
+	"reset the durability of {_item}",
+	"set durability of player's held item to 0"})
+@Since("1.2, INSERT VERSION (durability reversed)")
 public class ExprDurability extends SimplePropertyExpression<Object, Long> {
 
+	private boolean durability;
+
 	static {
-		register(ExprDurability.class, Long.class, "((data|damage)[s] [value[s]]|durabilit(y|ies))", "itemtypes/blocks/slots");
+		register(ExprDurability.class, Long.class, "(damage[s] [value[s]]|durability:durabilit(y|ies))", "itemtypes/slots");
```

Comment:
Add `[max]` to the durability part of the syntax as the method in Spigot is `getMaxDurability`

Reply:
There is an expression for that already [ExprMaxDurability](https://github.com/SkriptLang/Skript/blob/master/src/main/java/ch/njol/skript/expressions/ExprMaxDurability.java)

Reply:
@TheLimeGlass should I combine the two expressions? or just leave them be

Reply:
Well if the expression already exists, why is it being added here?

Reply:
There's already an expression for getting the max durability, this expression however, was the damage value, data value and durability (which acted the same as damage value)